klaviyo-swift-sdk.xcodeproj/project.pbxproj

```diff
@@ -22,6 +22,7 @@
 
 /* Begin PBXBuildFile section */
 		7C375BC028E77149008605B3 /* KlaviyoTestUtils.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7C375BBF28E77149008605B3 /* KlaviyoTestUtils.swift */; };
+		7C54A46F2902F01E00611E4E /* LoggerClient.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7C54A46E2902F01E00611E4E /* LoggerClient.swift */; };
```

Comment:
Question: Should we start gitignoring this file now that we are using SPM? Locals should be able to regenerate it with `swift package generate-xcodeproj` right?

Reply:
I'm not entirely sure how safe that is though...